doc/userguide/src/CreatingTestData/Variables.rst

```diff
@@ -1171,6 +1172,8 @@ Variables with the test case scope are visible in a test case and in
 all user keywords the test uses. Initially there are no variables in
 this scope, but it is possible to create them by using the BuiltIn_
 keyword :name:`Set Test Variable` anywhere in a test case.
+It is a fatal exception to call :name:`Set Test Variable` outside the
+scope of a test (e.g. in a Suite Setup or Teardown).
```

Comment:
Wording should be changed here as well. Either same style as above or something like this to avoid repetition:
> Calling :name:`Set Test Variable` in suite setup or teardown is an error.